src/main/java/frc/robot/RobotContainer.java

```diff
@@ -74,14 +76,38 @@ public class RobotContainer {
   /** Singleton instance of {@link UrMom} for the whole robot. */
   public static UrMom urMom = new UrMom();
 
+  /*
+   * *************
+   * * SUPPLIERS *
+   * *************
+   */
+  private Supplier<Double> scaledControllerLeftXAxisSupplier = () -> getScaledControllerLeftXAxis();
+  private Supplier<Double> scaledControllerLeftYAxisSupplier = () -> getScaledControllerLeftYAxis();
+  private Supplier<Double> scaledControllerRightXAxisSupplier =
+      () -> getScaledControllerRightXAxis();
+  private Supplier<Double> scaledControllerRightYAxisSupplier =
+      () -> getScaledControllerRightYAxis();
+  private Supplier<Boolean> povUpDirectionSupplier = () -> rawXboxController.getPOV() == 0;
+  private Supplier<Boolean> povRightDirectionSupplier = () -> rawXboxController.getPOV() == 90;
+  private Supplier<Boolean> povDownDirectionSupplier = () -> rawXboxController.getPOV() == 180;
+  private Supplier<Boolean> povLeftDirectionSupplier = () -> rawXboxController.getPOV() == 270;
```

Comment:
Maybe we move these down into `configureBindings()`? idk, thoughts?

Reply:
Just the POV suppliers, not the axis suppliers

Reply:
I think thats a good idea

Reply:
im a liar, dont do that